data/json/hobbies.json

```
@@ -90,6 +93,357 @@
+    "points": 4,
+    "skills": [ { "level": 2, "name": "melee" }, { "level": 3, "name": "unarmed" } ],
+    "traits": [ "MARTIAL_ARTS" ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "self_defense",
+    "name": "Self-Defense Classes",
+    "description": "You have taken some self-defense classes at a nearby gym.  You start with your choice of Boxing, Capoeira, Krav Maga, Muay Thai, Ninjutsu, Wing Chun, or Zui Quan.",
+    "points": 4,
+    "skills": [ { "level": 2, "name": "melee" }, { "level": 3, "name": "unarmed" } ],
+    "traits": [ "MARTIAL_ARTS2" ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "shaolin",
+    "name": "Shaolin Adept",
+    "description": "You have studied the arts of the Shaolin monks.  You start with one of the five animal fighting styles: Tiger, Crane, Leopard, Snake, or Dragon.",
+    "points": 4,
+    "skills": [ { "level": 2, "name": "melee" }, { "level": 3, "name": "unarmed" }, { "level": 3, "name": "dodge" } ],
+    "traits": [ "MARTIAL_ARTS3" ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "melee_training",
+    "name": "Melee Weapon Training",
+    "description": "You have practiced fighting with weapons.  You start with your choice of Eskrima, Fencing, Fior Di Battaglia, Medieval Swordsmanship, Niten Ichi-Ryu, Pentjak Silat, or Sōjutsu.",
+    "points": 5,
+    "skills": [
+      { "level": 2, "name": "melee" },
+      { "level": 3, "name": "bashing" },
+      { "level": 3, "name": "stabbing" },
+      { "level": 3, "name": "cutting" }
+    ],
+    "traits": [ "MARTIAL_ARTS5" ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "gourmand",
+    "name": "Gourmand",
+    "description": "You truly enjoy eating and can eat faster, can eat and drink more, than anyone else!",
+    "points": 3,
+    "traits": [ "GOURMAND" ],
+    "skills": [ { "level": 1, "name": "cooking" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "hates_books",
+    "name": "Hates Books",
+    "description": "Reading is for nerds!  Boring books are more boring, and you can't have fun by reading books.",
+    "points": -1,
+    "traits": [ "HATES_BOOKS" ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "hoarder",
+    "name": "Hoarder",
+    "description": "You don't feel right unless you're carrying as much as you can.  You suffer morale penalties for carrying less than maximum volume (weight is ignored).  Xanax can help control this anxiety.",
+    "points": -3,
+    "traits": [ "HOARDER" ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "nomadic",
+    "name": "Nomadic",
+    "description": "You're too adventurous for your own good.  The more time you spend somewhere, the unhappier it makes you to be there.",
+    "points": -1,
+    "traits": [ "NOMAD", "OUTDOORSMAN" ],
```

```suggestion
    "traits": [ "NOMAD" ],
```
We should be careful about rolling too many things together. A nomad might still not like getting wet.